docs/index.bs

```diff
@@ -3145,6 +3145,23 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Let |response| be |requestResponse|'s response.
                   1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                   1. Return |response|.
+          1. Else if |source| is {{RouterSource/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+
+              Note: In order to avoid duplicated fetches to |request|, the user agent may reuse |raceNetworkRequestResponse| as the result of |e|'s [=FetchEvent/potential response=], or the fallback request.
+
+              1. Let |raceFetchController| be null.
```

Comment:
Don't we need to abort |raceFetchController| when "[Dispatch](https://dom.spec.whatwg.org/#concept-event-dispatch) e at activeWorker’s [global object](https://w3c.github.io/ServiceWorker/#dfn-service-worker-global-object)." succeeds?

Reply:
Good catch. 

If the fetch handler result is fallback, we may want to reuse the inflight request as the fallback request.
If the fetch handler responded, the race fetch is not needed anymore, so we can abort it.

Reply:
I just wondered the case for race condition.
I suppose |response| can be overwritten by raceFetchController's response if raceFetchController's abort happened after getting response from it.

Even if |raceFetchController| side wins the competition, fetch event side seems not be aborted.  Is it fine to return |response| twice?

Reply:
I believe we should complete the fetch handler execution rather than abort it. But you're right, `Handle Fetch` shouldn't return the response twice.

Updated `Handle Fetch` to manage the race result with queue, which return |response| only once.

Reply:
Thanks for the update.  I think the race condition is resolved.